test/test_manager.py

```diff
@@ -165,6 +177,66 @@ def test_to_screen(manager):
     manager.c.prev_screen()
     assert manager.c.window.info()["name"] == "one"
 
+@multi_screen_manager_config
+def test_swap_screens(qtile):
+    self = qtile
```

Comment:
```suggestion
def test_swap_screens(manager):
```
`manager` is a defined fixture name. It needs to be used here. Please don't use the `self` alias here, that's inconsistenc across tests so those references need to be changed back to `manager`.